recipes/holopy/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "HoloPy" %}
+{% set version = "3.0_beta" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/manoharan-lab/holopy.git
+  git_rev: 0fbb2f08181ed4f5e951ff880faaec62e5905a5f
+
+build:
+  number: 3
+  skip: True  # [py2k or py34 or win]
```

Comment:
Ok.

> Regarding python skip: We only support python 3. I was getting a failure with numpy imports during the py35 build on circle or travis that only occurred when the py34 build preceded it. Either py34 or py35 works fine individually (see my earlier comments on this PR). Adding the numpy x.x may solve this?
>
> with numpy x.x, the py34 np111 build fails.

As @scopatz mentioned below, I think we have worked around this issue in PR ( https://github.com/conda-forge/staged-recipes/pull/2126 ). Seems to be a `conda-build-all` bug.